### ISSUE TITLE
Adding features to Iris_Classifier_using_Keras.ipynb

### ISSUE THREAD

**Issue by Halix267** [HarshCasper/NeoAlgo#285]
**Is your feature request related to a problem? Please describe.**
 The target column of the dataset have sequencial values and also the performance of the  model is depends on only one  test set.
**Describe the solution you'd like**
I am shuffling the values of iris dataset i.e both data and the target columns and then splitting into train and test sets.
And for evaluating the model performance i m using k_fold cross validation

**Describe alternatives you've considered**
when u are building the architecture of the model .. i would like to suggest that initialization of weights with he uniform method may increase yr n=model performance.
Also expanding the architecture of the model may increase the accuracy of the training set

**Additional context**

**Comment by Halix267** [comment]
I want to work on this project @HarshCasper

**Comment by Halix267** [comment]
I want to work on this issue @VijayaGB98

**Comment by vgb-codes** [comment]
These improvements are wonderful. @Halix267. Please feel free to add 1) Shuffling 2) K-fold Cross Validation and 3) Initialization of weights. Since the dataset was small, I demonstrated the use of DNN using only a few layers. If you find significant model improvement, you can add more layers. 

I'm assigning you this issue.

Also please don't 'assign' the issue to individuals that you would like to ping. '@username' would suffice.
edit: The assigning is happening automatically from our end. We will get it fixed shortly.

**Comment by mar-galho** [comment]
It's not his fault. We need to fix the templates - they are assigning us automatically...let me work on that now, actually

**Comment by vgb-codes** [comment]
@ricardoprins Oh then I apologize for my comment @Halix267. We will get it fixed ASAP.
